scripts/merges/do_merge.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
```

Comment:
You might want to drop these into `/scripts/ci/` as that folder is attributed to the releng team in CODEOWNERS, or alternatively add `/scripts/merges/` in CODEOWNERS.

Reply:
Good point, I'll switch to scripts/ci/